build/custom.mk

```diff
@@ -19,4 +25,4 @@ ifndef MM_RUDDER_WRITE_KEY
     MM_RUDDER_WRITE_KEY = 1d5bMvdrfWClLxgK1FvV3s4U1tg
 endif
 
-GO_BUILD_FLAGS += -ldflags '-X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderWriteKey=$(MM_RUDDER_WRITE_KEY)"'
+GO_BUILD_FLAGS += -ldflags '-X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderWriteKey=$(RUDDER_WRITE_KEY)"'
```

Comment:
Can we have the definition of `RUDDER_WRITE_KEY` be above this line, rather than at the top of the file?